api_tests/base/test_throttling.py

```diff
@@ -1,120 +1,130 @@
-import pytest
 import mock
-from nose.tools import *  # flake8: noqa
+import pytest
 
 from api.base.settings.defaults import API_BASE
-
-from tests.base import ApiTestCase
 from osf_tests.factories import AuthUserFactory, ProjectFactory
 
 
-pytestmark = pytest.mark.skip('Unskip when throttling no longer fails on travis')
+# pytestmark = pytest.mark.skip('Unskip when throttling no longer fails on travis')
```

Comment:
So does it work on work on travis now? If so, remove comment, else, probably needs to skip.